packages/bot-web-ui/src/stores/__tests__/self-exclusion-store.spec.tsx

```diff
@@ -0,0 +1,138 @@
+import SelfExclusionStore from '../self-exclusion-store';
+
+type TMockCore = {
+    client: {
+        is_eu: boolean;
+        is_virtual: boolean;
+        getSelfExclusion: () => void;
+        self_exclusion: {
+            max_losses?: number;
+        };
+    };
+};
+
+describe('SelfExclusionStore', () => {
+    let mockCore: TMockCore;
+
+    beforeEach(() => {
+        mockCore = {
+            client: {
+                is_eu: false,
+                is_virtual: false,
+                getSelfExclusion: jest.fn(),
+                self_exclusion: {},
+            },
+        };
+    });
+
+    it('Should initialize with default values', () => {
+        const store = new SelfExclusionStore(null, mockCore);
```

Comment:
@maryia-matskevich-deriv since this line is common for all the tests, can we just add this to the before each block at the top?

Reply:
✅done